native/cocos/editor-support/spine-wasm/spine-skeleton-instance.h

```diff
@@ -88,5 +102,5 @@ class SpineSkeletonInstance {
     spine::HashMap<spine::TrackEntry *, uint32_t> _trackListenerSet{};
     UserData _userData;
     spine::Vector<SpineDebugShape> _debugShapes{};
-    spine::HashMap<spine::Slot*, spine::String> _slotTextureSet{};
+    spine::Vector<SlotCacheInfo> _vecSlotTextures{};
```

Comment:
Please keep using spine::HashMap, fix the spine::HashMap issue and send a PR to spine runtime official repo .